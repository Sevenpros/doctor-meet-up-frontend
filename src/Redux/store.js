import { configureStore } from '@reduxjs/toolkit';
// import logger from 'redux-logger';
import doctorsReducer from './doctors/doctors';
import doctorReducer from './doctors/doctor';

const store = configureStore({
  reducer: {
    doctors: doctorsReducer,
    doctor: doctorReducer,
  },
  // middleware: (getDefaultMiddleware) => getDefaultMiddleware().concat(logger),
});
export default store;
