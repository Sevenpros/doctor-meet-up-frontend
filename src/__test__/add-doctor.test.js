import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../redux/store';
import AddDoctor from '../Components/Add_doctors';
import { BrowserRouter as Router } from 'react-router-dom';

describe('Add_doctor tests', () => {
  it('render Add_doctor test', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <Router>
        <AddDoctor />
        </Router>
       </Provider>,

    );
    expect(tree).toMatchSnapshot();
  });
  
});
