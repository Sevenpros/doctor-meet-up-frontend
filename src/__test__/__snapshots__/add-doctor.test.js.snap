// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add_doctor tests render Add_doctor test 1`] = `
<div
  className="add-doctor"
>
  <form
    className="form"
  >
    <div
      className="form-item"
    >
      <label
        className="form-label"
        htmlFor="first_name"
      >
        First Name
      </label>
      <input
        className="form-controls"
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <div
      className="form-item"
    >
      <label
        className="form-label"
        htmlFor="last_name"
      >
        Last Name
      </label>
      <input
        className="form-controls"
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <div
      className="form-item"
    >
      <label
        className="form-label"
        htmlFor="city"
      >
        City
      </label>
      <input
        className="form-controls"
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <div
      className="form-item"
    >
      <label
        className="form-label"
        htmlFor="phone"
      >
        Contact Number
      </label>
      <input
        className="form-controls"
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <div
      className="form-item"
    >
      <label
        className="form-label"
        htmlFor="email"
      >
        Email
      </label>
      <input
        className="form-controls"
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <div
      className="form-item"
    >
      <label
        className="form-label"
        htmlFor="bio"
      >
        Bio
      </label>
      <input
        className="form-controls"
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <div
      className="form-item"
    >
      <label
        className="form-label"
        htmlFor="specialization"
      >
        Specialization
      </label>
      <input
        className="form-controls"
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <div
      className="form-item"
    >
      <label
        className="form-label"
        htmlFor="photo"
      >
        Photo
      </label>
      <input
        className="form-controls"
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <button
      className="book-btn"
      onClick={[Function]}
      type="button"
    >
      Add Doctor
    </button>
  </form>
  <div
    className="Toastify"
  />
</div>
`;
